kafka_fdw--0.0.2--0.0.3.sql

```diff
@@ -0,0 +1,22 @@
+DROP FUNCTION kafka_get_watermarks(IN regclass, OUT int, OUT int, OUT int);
+
+CREATE FUNCTION kafka_get_watermarks(
+    IN rel regclass,
+	OUT partition int,
+	OUT offset_low bigint,
+	OUT offset_high bigint)
+RETURNS SETOF record
+AS 'MODULE_PATHNAME', 'kafka_get_watermarks'
+LANGUAGE C STRICT;
+
+DO $$
+DECLARE version_num INTEGER;
+BEGIN
+    SELECT current_setting('server_version_num') INTO STRICT version_num;
+    IF version_num > 90600 THEN
+        EXECUTE 'ALTER FUNCTION kafka_get_watermarks(regclass) PARALLEL SAFE';
```

Comment:
mmh is that really `PARALLEL SAFE` ? 
The function is definitely not IMMUTABLE and potentially returns different results for different parallel workers. 
I'd probably not mark them as such

Reply:
Even though i don't think anything bad could happen when it's being called from parallel processes, there is not much sense in doing so. So I'll be happy to remove `PARALLEL SAFE` if nobody minds.

Reply:
I mean it doesn't change the state of kafka topic

Reply:
Sure but still different workers would return different offsets and that might be concerning. But tbh I'm not 100% in the topic which conditions must meet to make a function parallel safe.

Reply:
My understanding is that parallel safe function is one that doesn't have side effects, i.e. doesn't change some global state (e.g. inserting into kafka) and doesn't change shared state without ensuring consistency (i.e. protection from race conditions).
Though I agree that we could remove parallel safety label because it hardly makes sense to use the function in parallel context.

Reply:
done